compiler/modules/front/symbols.bal

```diff
@@ -11,6 +12,7 @@ type ModuleSymbols record {|
     map<Import>[] partPrefixes = [];
     t:Context tc;
     boolean allowAllTypes = false;
+    [t:SemType, d:Location][] possiblyEmptyTypes = [];
```

Comment:
I think the name should better capture the idea that these are types that must be checked for non-emptiness.

Reply:
renamed to `deferredEmptinessChecks`